sections/futures/ProfitCalculator/PnLs.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { wei } from '@synthetixio/wei';
+
+import StatWithContainer from './StatWithContainer';
+import { PositionSide } from '../types';
+
+type PnLsProps = {
+	stopLoss: string;
+	exitPrice: string;
+	entryPrice: string;
+	amountInAsset: string;
+	leverageSide: PositionSide;
+};
+
+const PnLs: React.FC<PnLsProps> = ({
+	stopLoss,
+	exitPrice,
+	entryPrice,
+	amountInAsset,
+	leverageSide,
+}) => {
+	let rateOfReturn: any = 0,
+		profit: any = 0,
+		loss: any = 0;
+
+	// Calculate values for each stat
+	if (entryPrice !== '' && exitPrice !== '' && stopLoss !== '' && amountInAsset !== '') {
```

Comment:
I'd prefer if this didn't check _every_ input for _every_ calculation and instead checks the necessary input for that calculation.

For example, if I have entry price, exit price, and size it should calculate "Exit PnL", but leave the "Stop PnL" blank.

<img width="708" alt="image" src="https://user-images.githubusercontent.com/10401554/165139775-e83f9d6a-f75b-4ed9-b07d-266331b09b05.png">